packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -161,6 +176,30 @@ contract DeployConfig is Script {
         useInterop = _readOr(_json, "$.useInterop", false);
     }
 
+
+    function fork() public view returns (Fork fork_) {
+        // let env var take precedence
+        fork_ = Config.fork();
+        if (fork_ == Fork.NONE) {
+            // Will revert if no deploy config can be found either.
+            fork_ = latestGenesisFork();
+            console.log("DeployConfig: using deploy config fork: %s", fork_.toString());
+        } else {
+            console.log("DeployConfig: using env var fork: %s", fork_.toString());
+        }
+    }
+
+    function latestGenesisFork() public view returns (Fork) {
```

Comment:
Thoughts on having just 1 public function vs 2?

Reply:
Even though `latestGenesisFork` is not used publicly right now, I left it public because it conveys a different information about the deploy config that could be used in the future by another contract. I thought it doesn't really matter. I will make it internal and if we ever need it as a public fn, can make it public again.